daprdocs/content/en/reference/components-reference/supported-state-stores/setup-postgresql-v2.md

```diff
@@ -0,0 +1,165 @@
+| `timeout` | N | Timeout for operations on the database, as a [Go duration](https://pkg.go.dev/time#ParseDuration). Integers are interpreted as number of seconds. Defaults to `20s` | `"30s"`, `30` |
+| `cleanupInterval` | N | Interval, as a Go duration or number of seconds, to clean up rows with an expired TTL. Default: `1h` (1 hour). Setting this to values <=0 disables the periodic cleanup. | `"30m"`, `1800`, `-1` |
+| `maxConns` | N | Maximum number of connections pooled by this component. Set to 0 or lower to use the default value, which is the greater of 4 or the number of CPUs. | `"4"` |
+| `connectionMaxIdleTime` | N | Max idle time before unused connections are automatically closed in the connection pool. By default, there's no value and this is left to the database driver to choose. | `"5m"` |
+| `queryExecMode` | N | Controls the default mode for executing queries. By default Dapr uses the extended protocol and automatically prepares and caches prepared statements. However, this may be incompatible with proxies such as PGBouncer. In this case it may be preferrable to use `exec` or `simple_protocol`. | `"simple_protocol"` |
+| `actorStateStore` | N | Consider this state store for actors. Defaults to `"false"` | `"true"`, `"false"` |
+
+## Setup PostgreSQL
+
+{{< tabs "Self-Hosted" >}}
+
+{{% codetab %}}
+
+1. Run an instance of PostgreSQL. You can run a local instance of PostgreSQL in Docker with the following command:
+
+     ```bash
+     docker run -p 5432:5432 -e POSTGRES_PASSWORD=example postgres
+     ```
+
+     > This example does not describe a production configuration because it sets the password in plain text and the user name is left as the PostgreSQL default of "postgres".
+
+2. Create a database for state data.  
+    Either the default "postgres" database can be used, or create a new database for storing state data.
+
+    To create a new database in PostgreSQL, run the following SQL command:
+
+    ```sql
+    CREATE DATABASE my_dapr;
+    ```
+  
+{{% /codetab %}}
+
+{{% /tabs %}}
+
+## Advanced
+
+### Differences between v1 and v2
+
+The PostgreSQL state store v2 was introduced in Dapr 1.13. The pre-existing v1 remains available and is not deprecated.
+
+In the v2 component, the table schema has been changed significantly, with the goal of increasing performance and reliability. Most notably, the value stored by Dapr is now of type _BYTEA_, which allows faster queries and in some cases it's more space-efficient than the previously-used _JSONB_ column.  
+On the downside, because of the change above, the v2 component does not support the Dapr state store query APIs.
+
+Additionally, etags are now random UUIDs, which ensures better compatibility with other PostgreSQL-compatible databases such as CockroachDB.
+
+Because of the changes, it is not possible for v1 and v2 components to read or write data from the same table. At this stage, it is also not possible to migrate data between the two versions of the component.
```

Comment:
```suggestion
Because of these changes, v1 and v2 components are not able to read or write data from the same table. At this stage, it's also impossible to migrate data between the two versions of the component.
```